Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -606,6 +642,10 @@ BoardInit (
       VariableConstructor (VarBase, VarSize);
     }
     SaveOtgRole();
+    if (PcdGetBool (PcdSplashEnabled)) {
```

Comment:
Framebuffer console output may be turned on while splash is disabled. In this case I think we should also enable WC. Can we add that to this condition? Or perhaps a new generic PCD which is simple "Framebuffer Enabled"

Reply:
That is a good point.  I think it is better to add a new PCD to control if the GFX should be initialized by bootloader.  Then we can use this PCD to control.